types/eslint/rules/possible-errors.d.ts

```diff
@@ -355,6 +355,20 @@ export interface PossibleErrors extends Linter.RulesRecord {
      */
     "no-obj-calls": Linter.RuleEntry<[]>;
 
+    /**
+     * Rule to disallow returning values from Promise executor functions.
+     *
+     * @since 7.3.0
+     * @see https://eslint.org/docs/rules/no-promise-executor-return
+     */
+    "no-promise-executor-return": Linter.RuleEntry<
+        [
+            Partial<{
```

Comment:
It seems there is no need use `Partial` util, `?` is cleaner enough.

Reply:
I would agree, except that I have not found any other rule typing that uses `?` instead of `Partial`. This holds even for rules that have only a single property in their configuration object.

Reply:
I should add a `@default` tag though.

Reply:
d67ddc1b470d8641dada4d54c0f542e1bc0e2ac7

Reply:
Well, those are added before my review.

Reply:
As you wish c08833e67275a04010fc2381cf0fe19f4898365e ... simpler might indeed be better here.